### ISSUE TITLE
SelectCharSizeHeight and SelectCharSizeWidth must be combined?

### ISSUE THREAD

**Issue by bookhanming** [igorocampos/ESCPOS#10]
Hi igorocampos, I noticed your these two commands share the same GS code.

```
SelectCharSizeHeight(CharSizeHeight charSize)
SelectCharSizeWidth(CharSizeWidth charSize)
```

`0x1D, 0x21, (byte)charSize`

When I tried to SelectCharSizeHeight and SelectCharSizeWidth one after another, the previous `charSize` is lost.

Please see this photo: (I set it to 7(Octuple) for Height and 112(Octuple) for Width with two separate commands)
Effectively, the two commands are: 
`0x1D, 0x21, 7`
`0x1D, 0x21, 112`
![WhatsApp Image 2020-01-16 at 8 30 29 PM](https://user-images.githubusercontent.com/55100040/72525618-888b2f80-389f-11ea-8f12-dad6e9df97a3.jpeg)

However, when I combine the value 112+7 as 119, with this single command:
`0x1D, 0x21, 119`

Then only I get the correct 8x + 8x character size:
![WhatsApp Image 2020-01-16 at 8 24 11 PM](https://user-images.githubusercontent.com/55100040/72525889-1c5cfb80-38a0-11ea-9d8e-b1e5788c3497.jpeg)

I tried to workaround with your library by ORing the CharSizeWidth and CharSizeHeight but cannot, because the argument type is either one.

Please look into this... Have a good day!

**Comment by igorocampos** [comment]
Hello, you are right again. However, if I remember it correctly that was done in purpose, so one could only use one of the modifiers.

I guess that it can be changed, as you said. Would you like to propose a Pull Request for that?

**Comment by bookhanming** [comment]
Thanks for your prompt reply. Yes, I would like to propose a Pull Request for that. But how do I do that? (I am new to GitHub actually)

**Comment by igorocampos** [comment]
Ok, so the first thing you need to do is fork my repository. So you will have a bookhanming/ESCPOS repository of your own.

Then you can change whatever you need and push it to a branch in *your* forked repository.

After that you come back here in my repository, click in Pull Requests> New Pull Request  and in the link `compare across forks`.

You will see that the combobox below will change so you can select on the left one my master branch (of this repository) and on the right *your* branch with the updates in your repository.

**Comment by igorocampos** [comment]
fixed by #11
